src/main/scala/mesosphere/marathon/core/task/tracker/TaskTrackerConfig.scala

```diff
@@ -0,0 +1,17 @@
+package mesosphere.marathon.core.task.tracker
+
+import org.rogach.scallop.ScallopConf
+
+trait TaskTrackerConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  lazy val taskTrackerRequestTimeout = opt[Int]("task_tracker_request_timeout",
```

Comment:
can we rename this parameter to internalXXX so that we easily see it is internal and hidden by default?